libdnf/repo/Repo.cpp

```diff
@@ -610,6 +621,9 @@ std::unique_ptr<LrHandle> Repo::Impl::lrHandleInitRemote(const char *destdir)
     if (!conf->proxy_username().empty()) {
         userpwd = conf->proxy_username().getValue();
         if (!userpwd.empty()) {
+            if (conf->proxy_password().empty()) {
+                throw RepoError(tfm::format(_("repo '%s': 'proxy_password' is not set"), id));
```

Comment:
What about to add something more. Like `_("repo '%s': proxy_username is set but not 'proxy_password'")`? What do you think?

Reply:
OK. I will change it.

Reply:
Done.